### PR TITLE
Docs: Correct link to Iframe origin policy

### PR DESCRIPTION
**Pull request by bpaduch**
Minor - corrected link to doc.

to: @Meggin

### CODE REVIEW

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ To enable this, copy the file [remote.html](../../3p/remote.html) to your web se
 <meta name="amp-3p-iframe-src" content="https://assets.your-domain.com/path/to/remote.html">
 ```
 
-The `content` attribute of the meta tag is the absolute URL to your copy of the remote.html file on your web server. This URL must use a "https" schema. It is not allowed to reside on the same origin as your AMP files. E.g. if you host AMP files on "www.example.com", this URL must not be on "www.example.com" but e.g. "something-else.example.com" is OK. See the doc ["Iframe origin policy"](../spec/amp-iframe-origin-policy.md) for further details on allowed origins for iframes.
+The `content` attribute of the meta tag is the absolute URL to your copy of the remote.html file on your web server. This URL must use a "https" schema. It is not allowed to reside on the same origin as your AMP files. E.g. if you host AMP files on "example.com", this URL must not be on "example.com" but e.g. "something-else.example.com" is OK. See the doc ["Iframe origin policy"](../../spec/amp-iframe-origin-policy.md) for further details on allowed origins for iframes.
```
Comment (jridgewell) [inline extensions/amp-ad/amp-ad.md]: Why were the `www`s removed?

Comment (bpaduch) [inline extensions/amp-ad/amp-ad.md]: Because they were creating active links in the rendered doc; which I thought was deceiving and unncessary (?).

Comment (jridgewell) [inline extensions/amp-ad/amp-ad.md]: Wrap them in backticks instead of quotes, then.

Comment (bpaduch) [inline extensions/amp-ad/amp-ad.md]: Done

**Comment by bpaduch** [comment]
I signed it!

**Pull request merged**